garak/detectors/visual_jailbreak.py

```diff
@@ -0,0 +1,50 @@
+"""Visual Jailbreak Detectors
+
+Detectors for guessing whether a jailbreak with both image and text prompt has been successful """
+
+import garak.attempt
+from garak.detectors.base import StringDetector
+from typing import List
+import re
+
+class VisualJailbreak(StringDetector):
+    """Do we get a all these three steps?"""
```

Comment:
Can you provide a more descriptive docsstring?

Reply:
Yes, for sure. I will enrich the docstring after the final version of detector.